mongomock/collection.py

```diff
@@ -518,7 +518,7 @@ def save(self, to_save, manipulate = True, safe = False, **kwargs):
                         manipulate, safe, _check_keys = True, **kwargs)
             return to_save.get("_id", None)
 
-    def remove(self, spec_or_id = None, search_filter = None):
+    def remove(self, spec_or_id = None, search_filter = None, **kwargs):
```

Comment:
I don't like receiving kwargs in such cases. It would make production code passing nonexistent parameters pass tests, but fail in production